__tests__/app.test.js

```diff
@@ -313,10 +429,22 @@ describe('9 - GET /api/users', () => {
           expect(user).toMatchObject({
             username: expect.any(String),
             name: expect.any(String),
-            avatar_url: expect.any(String)
+            avatar_url: expect.any(String),
           });
         });
       });
   });
 });
 
+describe('11 - DELETE /ap/comments/comment_id', () => {
+  test.only('should delete the comment with the comment_id', () => {
+    return request(app)
+        .delete(`/api/comments/3`)
+        .expect(204)
+  });
+  test('should return 404 when an invalid comment_id is passed', () => {
+    return request(app)
+        .delete(`/api/comments/100`)
+        .expect(404)
+  });
+});

```

Comment:
Another test would be handy. What if the comment id is invalid? eg. `/api/comments/banana`